### DISCUSSION TITLE
Welcome to EJBCA Discussions!

### DISCUSSION THREAD

**Discussion by KarolinHem** [Keyfactor/ejbca-ce#28] [Announcements]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we
- build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by primetomas** [comment]
I'm Tomas, I started EJBCA back in 2001, still pushes some pieces of code occasionally, answers questions and listen to feedback. Basically I'm here to help!

**Comment by mike-agrenius-kushner** [comment]
I'll go too! 

Name's Mike, one of the long time EJBCA devs, Product Owner for the best part of six years or so and currently a product architect with Keyfactor. Still regularly helping out with bugs, adding minor features and generally helping the teams.

**Comment by minhtrienle** [comment]
Hi,
Currently I am installing EJBCA 7.4.3.2 with wildfly 10.1.0.final when when I connect to [standalone@localhost:9990] after I enter the command
/subsystem=undertow/server=default-server/https-listener=httpspriv:add(socket-binding=httpspriv, security-realm="SSLRealm", verify-client=REQUIRED)
then there is an error
{
"outcome" => "failed",
"failure-description" => {"WFLYCTL0180: Services with missing/unavailable dependencies" => undefined},
"rolled-back" => true
}
Can you help me fix this error?
Sincerely thank you very much
I watch video tutorials at:
https://www.youtube.com/watch?v=e65A7-LuULU

**Reply by minhtrienle** [reply]
I have successfully fixed the error, if anyone has an error, please comment below, I will support.

**Comment by primetomas** [comment]
That's great to hear. For next time you can create a new discussion for every issue. Just click the green "New discussion" button if you go to Discussions home. Every issue/topic should have it's own thread so it is easily findable and can be closed/solved.
